src/components/search/styles.css

```diff
@@ -24,12 +24,24 @@
 
 .search-root .hit {
   @apply flex items-center;
+  /* Avoid having search results displayed on multiple lines */
+  height: 2em;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
I think this is `.truncate` in tailwinds... worth searching the docs

Reply:
yes it is :)